pkgs/top-level/python-packages.nix

```diff
@@ -4545,6 +4545,8 @@ in {
 
   pamqp = callPackage ../development/python-modules/pamqp { };
 
+  pantalaimon = callPackage ../applications/networking/instant-messengers/pantalaimon { };
+
```

Comment:
please move this to all-packages.nix and do:
```
pantalaimon = python3Packages.callPackage ../applications/networking/instant-messengers/pantalaimon { };
```

Reply:
generally, python-packages are for modules which are going to be re-used, applications go into all-packages

Reply:
whoops, of course. thanks :)

(I wonder if this knowledge could be encoded in CI tests)